doc/flame_svg/install.md

```diff
@@ -0,0 +1,8 @@
+# Installation
+
+
+## Use of the package
+
+It's possible to use this package in two way: as a **library** or **importing it**
```

Comment:
Not sure what you are referring to here, flame_svg vs svg? Both are libraries.

Reply:
You didn't reply to this, you just marked it as resolved?

Reply:
I edited the title as Use of the flame_svg package

Reply:
This sentence is still willing though, I don't think this file gives anything anyways

Reply:
I will remove it!